deployer/cloudflaredeployer.go

```diff
@@ -0,0 +1,281 @@
+package deployer
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/pkg/errors"
+
+	"github.com/suborbital/atmo/atmo/appsource"
+	"github.com/suborbital/atmo/atmo/options"
+	"github.com/suborbital/atmo/directive"
+	"github.com/suborbital/subo/project"
+	"github.com/suborbital/subo/subo/util"
+)
+
+const (
+	cloudflareDeployJobType = "cloudflare"
+	routerScriptName        = "subo-router"
+)
+
+//go:embed cloudflare/templates/request_worker.js
```

Comment:
All of these template files should be put into the `templates` directory and the template renderer should be used to write them into the `.deploy` directory when they need to be used (which is a Subo norm)

Reply:
done